bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifier.scala

```diff
@@ -12,38 +12,22 @@ import uk.ac.wellcome.platform.archive.common.bagit.services.BagReader
 import uk.ac.wellcome.platform.archive.common.ingests.models.IngestID
 import uk.ac.wellcome.platform.archive.common.storage.models._
 import uk.ac.wellcome.storage.listing.Listing
+import uk.ac.wellcome.storage.s3.S3ObjectLocationPrefix
 import uk.ac.wellcome.storage.{Location, Prefix}
 
 import scala.util.Try
 
-trait StandaloneBagVerifier[BagLocation <: Location, BagPrefix <: Prefix[
-  BagLocation
-]] extends BagVerifier[
-      StandaloneBagVerifyContext[BagPrefix],
-      BagLocation,
-      BagPrefix
-    ] {
-  override def verifyReplicatedBag(
-    root: StandaloneBagVerifyContext[BagPrefix],
-    space: StorageSpace,
-    externalIdentifier: ExternalIdentifier,
-    bag: Bag
-  ): Either[BagVerifierError, Unit] = Right(())
-}
-
 trait ReplicatedBagVerifier[
```

Comment:
minor: is it worth splitting ReplicatedBagVerifier out into its own file, since S3StandaloneBagVerifier now lives elsewhere?